cooperative_lanechange/src/cooperative_lanechange.cpp

```diff
@@ -794,7 +818,96 @@ namespace cooperative_lanechange
 
     }
 
-     lanelet::BasicLineString2d CooperativeLaneChangePlugin::create_lanechange_path(lanelet::ConstLanelet& start_lanelet,lanelet::ConstLanelet& end_lanelet)
+        for (const auto& p : points)
+        {
```

Comment:
This for loop is defined outside the above function, and break the class structure.